content/howto/geodesic/iam-creds-authentication.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Authenticate with AWS inside Geodesic"
```

Comment:
Seems like this should be more specific named "Authenticate with AWS inside Geodesic using `aws-vault`"

and maybe the filename `authenticate-with-aws-vault.md`

Reply:
Yeah good call, will update.

Reply:
Updated.